### DISCUSSION TITLE
Can't play on Netplay

### DISCUSSION THREAD

**Discussion by Dolanquac** [mupen64plus-ae/mupen64plus-ae#1165] [Q&A]
Tried playing Mario Party 2 with friends using Netplay, but the game didn't started (we couldn't even hear the game, just watching a black screen with no sound). We tried to play Mario Kart 64 and it also didn't work, then Super Smash Bros and, yeah, didn't work. What happened with Netplay servers?

**Comment by fzurita** [comment]
Hi, the netplay servers are only for the initial connection. They are out of the loop once the "start game" option is selected.

I have heard of reports that if you have orientation locked, you only get a black screen. Can you go to settings then display and change the orientation to auto and see if that works?
